docs/timeseries/index.rst

```diff
@@ -310,8 +320,6 @@ Let's take a look at the final result:
    plt.xlabel('Time (days)')
    plt.ylabel('Normalized flux')
 
-It looks like there might be a hint of a secondary transit!
```

Comment:
The issue is that because of the way I'm getting the periodogram results now the transit ends up at the center of the plot, so the secondary transit is not as visible. Since I wasn't even 100% sure it was real anyway I'd rather remove.